### PR TITLE
finished displaying the reviews on page and added additional styling

### PR DESCRIPTION
**Pull request by zarina-m**
finished all styling on restaurant results along with displaying the results 
added a loading icon b/w the hover and review return

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -128,6 +123,48 @@ restaurantsApp.showMore = function(result){
     });
 }
 
+restaurantsApp.handleHover = function() {
+    $('.imageWrapper').off('mouseover').on('mouseover', async function(e) {
```
Comment (Mila-ashtari) [inline script.js:130]: what does .off() do?

Comment (zarina-m) [reply script.js:130]: if you are unfamiliar with .off() you can read about it here:
https://api.jquery.com/off/

we need to reset the event handlers every time more results are appended to the page. if we do not do this, another set of event handlers will be attached every time more results load

File: script.js [hunk 2]
```diff
@@ -128,6 +123,48 @@ restaurantsApp.showMore = function(result){
...
+            // We have already searched for this review, simply use it
+            reviewHtml = restaurantsApp.reviews[businessID];
+        } else {
+            let reviewInfo = await restaurantsApp.getReview(businessID);
+            if (reviewInfo.reviews.length > 0) {
+                // If there are any reviews, get the raint and text of the first review
+                let ratingText = reviewInfo.reviews[0].text;
+                let rating = reviewInfo.reviews[0].rating; 
+                console.log(rating);
+                console.log(parseInt(rating));
```
Comment (Mila-ashtari) [inline script.js]: these two can be removed

**Review by Mila-ashtari — COMMENTED** [review]
great, please resolve the conflicts as well

**Pull request merged**